user/models.py

```diff
@@ -1,3 +1,7 @@
 from django.db import models
 
-# Create your models here.
+class User(models.Model):
+    id = models.CharField(max_length=255, primary_key=True)
```

Comment:
This seems a remarkably peculiar choice for a primary key.

What is generating these strings you'll be using for the values?

Reply:
In other words, why isnt this an integer type,

Reply:
Or omit an explicit key entirely, and get the AutoField django will give you by default.